toqito/channel_props/is_extremal.py

```diff
@@ -0,0 +1,96 @@
+"""Module for checking whether a quantum channel is extremal."""
+
+import numpy as np
+from numpy.linalg import matrix_rank
+
+from toqito.channel_ops.choi_to_kraus import choi_to_kraus
+
+
+def _convert_to_kraus(channel):
+    r"""Convert a quantum channel into its Kraus representation.
+
+    The input channel can be provided as:
+      - A list of Kraus operators (each a numpy.ndarray),
+      - A Choi matrix (numpy.ndarray),
+      - A dictionary with a 'kraus' or 'choi' key,
+      - An object with a callable kraus() method.
+
+    Parameters
```

Comment:
We use a different style for specifying params etc. Take a look at the contributing guide in the documentation.